qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -64,24 +67,65 @@ def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwar
             # if we don't delete, then this value clashes with our quantum kernel
             del kwargs["kernel"]
 
-        self._quantum_kernel = quantum_kernel if quantum_kernel else QuantumKernel()
-
         if "random_state" not in kwargs:
             kwargs["random_state"] = algorithm_globals.random_seed
 
-        super().__init__(kernel=self._quantum_kernel.evaluate, *args, **kwargs)
+        # Class Fields
+        self._quantum_kernel = quantum_kernel or QuantumKernel()
+        self._kernel_trainer = kernel_trainer
+
+        # SVC constructor
+        super().__init__(kernel=self._quantum_kernel.evaluate, **kwargs)
 
     @property
     def quantum_kernel(self) -> QuantumKernel:
         """Returns quantum kernel"""
         return self._quantum_kernel
 
     @quantum_kernel.setter
-    def quantum_kernel(self, quantum_kernel: QuantumKernel):
+    def quantum_kernel(self, quantum_kernel: QuantumKernel) -> None:
         """Sets quantum kernel"""
+        # QuantumKernel passed
         self._quantum_kernel = quantum_kernel
         self.kernel = self._quantum_kernel.evaluate
 
+    @property
+    def kernel_trainer(self) -> Optional[QuantumKernelTrainer]:
+        """Returns quantum kernel trainer"""
+        return self._kernel_trainer
+
+    @kernel_trainer.setter
+    def kernel_trainer(self, qk_trainer: Optional[QuantumKernelTrainer]) -> None:
+        """Returns quantum kernel trainer"""
+        self._kernel_trainer = qk_trainer
+
+    def fit(
+        self, X: np.ndarray, y: np.ndarray, sample_weight: Optional[Sequence[float]] = None
+    ) -> SVC:
+        # Ensure a backend was specified
+        if self._quantum_kernel.quantum_instance is None:
+            raise QiskitMachineLearningError(
+                """
+            Error fitting QSVC. No quantum instance was specified.
+            """
+            )
+
+        # Ensure there are no unbound user parameters
```

Comment:
Reading the message in the exception answers the question why and in that regard this comment could be improved to be more useful to someone new to the code reading it.